src/DataDepsGenerators.jl

```diff
@@ -190,4 +282,10 @@ function mainpage_url(repo::DataRepo, dataname)
     getpage(url), url
 end
 
+struct GeneratorError <: Exception
+    repo::DataDepsGenerators.DataRepo
+end
+
+Base.showerror(io::IO, e::GeneratorError) = print(io, e.repo, " generator did not work")
```

Comment:
I feel better that `did not work`,
would be `was not suitable`.
Since this is the error thrown if it is detected that it could not be used, e.g via not being a DOI.
Obviously every error means that it did not work

Or if you add a `message` field, then it would be `was not suitable. $(e.message)`